app/App.tsx

```diff
@@ -70,8 +70,7 @@ export const App = () => {
     return history.listen(loc => {
       const page = loc.pathname + loc.search;
       ReactGA.set({ page });
-      ReactGA.pageview(loc.pathname);
-      window.scrollTo(0, 0); // TODO Is this necessary anymore?
```

Comment:
I couldn't for the life of me think why the above line would be necessary, so I removed it and didn't notice any side effects.